tests/test_lock.py

```diff
@@ -125,6 +125,34 @@ def test_extending_lock_no_longer_owned_raises_error(self, r):
         with pytest.raises(LockNotOwnedError):
             lock.extend(10)
 
+    def test_reacquire_lock(self, r):
+        lock = self.get_lock(r, 'foo', timeout=10)
+        assert lock.acquire(blocking=False)
+        time.sleep(1)
```

Comment:
Instead of sleeping (and slowing down the test suite) could we manually `PEXPIRE foo 1000` after acquiring the lock initially? Then after the `reacquire` we should see `pttl foo` back in the 9000-10000 range.